config/default/manager_webhook_patch.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: odh-model-controller
+  namespace: system
+#  labels:
+#    app.kubernetes.io/name: odh-model-controller
```

Comment:
I omitted to have the manifests as same as possible with respect before the migration, to prevent any unexpected side effects.

Reply:
not sure if I understood you.

Reply:
In the past we have had some upgrade issues because of adding/removing labels. Quite likely, it doesn't hurt to add labels here in the Deployment. But, IIRC, kustomize may also update the label selectors in `Service`. I think the `Service` needs to be re-created to change its label selectors, and our opendatahub-operator is not prepared for such change.

That's what I remember... I may not be remembering correctly. In case that's not the case, I still remember some issue close to that idea.

So, I preferred keeping the labels and annotations that we currently have.

Reply:
oh, ok, that makes sense.
Thanks